jobs/rocoto/wavepostsbs.sh

```diff
@@ -11,7 +11,7 @@ status=$?
 
 export job="wavepostsbs"
 export jobid="${job}.$$"
-
+export FORECAST_HOUR=$(( 10#${FHR3} ))
```

Comment:
This script won't run in operations. Consider putting this `FORECAST_HOUR` variable settings lower down in the JJOB or `scripts/` script.